docs/querying/api.md

```diff
@@ -679,7 +679,8 @@ URL query parameters:
 - `rule_name[]=<string>`: only return rules with the given rule name. If the parameter is repeated, rules with any of the provided names are returned. If we've filtered out all the rules of a group, the group is not returned. When the parameter is absent or empty, no filtering is done.
 - `rule_group[]=<string>`: only return rules with the given rule group name. If the parameter is repeated, rules with any of the provided rule group names are returned. When the parameter is absent or empty, no filtering is done.
 - `file[]=<string>`: only return rules with the given filepath. If the parameter is repeated, rules with any of the provided filepaths are returned. When the parameter is absent or empty, no filtering is done.
-
+- `match[]=<label_selector>`: Repeated label selector argument that selects rules which have configured labels that satisfy the selectors. Note that for alerting rules it matches the top-level non-templated labels, not the rendered labels of active alerts (instances). Optional.
```

Comment:
should we adjust the doc as well?

Reply:
Done in https://github.com/prometheus/prometheus/pull/10194/commits/711dbc1041c53e316679a3727dccabaef5474163. Updated and changed to make it consistent with rest of the params.

Reply:
ty!